app/experimenter/experiments/models.py

```diff
@@ -505,21 +505,32 @@ def is_paused_published(self):
 
     @property
     def monitoring_dashboard_url(self):
-        start_date = (self.start_date or datetime.date.today()) - datetime.timedelta(
-            days=1
-        )
+        if not self.is_rollout:
+            start_date = (self.start_date or datetime.date.today()) - datetime.timedelta(
+                days=1
+            )
 
-        if self.end_date:
-            end_date = self.end_date + datetime.timedelta(days=2)
-        else:
-            # add a day to account for Looker data being in UTC
-            end_date = datetime.date.today() + datetime.timedelta(days=1)
+            if self.end_date:
+                end_date = self.end_date + datetime.timedelta(days=2)
+            else:
+                # add a day to account for Looker data being in UTC
+                end_date = datetime.date.today() + datetime.timedelta(days=1)
 
-        return settings.MONITORING_URL.format(
-            slug=self.slug,
-            from_date=start_date.strftime("%Y-%m-%d"),
-            to_date=end_date.strftime("%Y-%m-%d"),
-        )
+            return settings.MONITORING_URL.format(
+                slug=self.slug,
+                from_date=start_date.strftime("%Y-%m-%d"),
+                to_date=end_date.strftime("%Y-%m-%d"),
+            )
+
+    @property
+    def rollout_monitoring_dashboard_url(self):
+        if self.is_rollout and (
+            self.status
+            == (NimbusExperiment.Status.LIVE or NimbusExperiment.Status.COMPLETE)
+        ):
```

Comment:
Yes exactly 👍

Reply:
Oh wait what, does that work?  I've never seen that syntax before....  I would've written it as

`self.status in (LIVE, COMPLETE)`

Reply:
Where did you get that syntax?  I don't think it works like that?

```py
ipdb> 1 == (1 or 2)
True
ipdb> 2 == (1 or 2)
False
```

Reply:
Another thing to think about is parametrically testing both statuses would probably catch that this doesn't work as intended.

Reply:
DANG IT. Python got me there.